views/lib/components/modal-form.hbs

```diff
@@ -9,15 +9,15 @@
                 {{#block "fields"}}{{/block}}
             </div>
             <div class="modal-footer">
-                <button type="button" class="btn btn-secondary btn-close" data-testid="btn-cancel-{{title}}">
+                <button type="button" class="btn btn-secondary btn-close" data-testid="btn-cancel">
```

Comment:
It look from code like that in past different cancel and submit actions to same time are avaible. You are sure and test on all places that this is not possible anymore?

Reply:
On sc-client i don't see anywhere where it is used, in e2e tests it is used in one place (for forgetting the password flow) and even in there is selector for only "btn-cancel-" -> pageLoginManagement.js l:9
So maybe it would be better to just delete this unnecessary dynamic part in data-testid and fix selector

Reply:
I spoke to Uzair everything is ok for him with "btn-cnacle- ".  We checked all the tests, it is used in 2 tests in this and one more (forgotPasswordDialog.feature)  where for Uzair this change is ok.